code/style.css

```diff
@@ -1,7 +1,335 @@
+  z-index: -1;
+}
+
+.header-desktop-video {
+  display: none;
+}
+
+main {
+  width: 100%;
+}
+/* accordion */
+.accordion-box {
+  max-height: 140vh;
+  height: auto;
+  padding: 2rem 0;
+}
+
+.accordion {
+  max-width: 80%;
+  width: auto;
+  margin: 2rem auto;
+  background-color: rgb(249, 249, 251);
+  border-radius: 2rem;
+  box-shadow: 0.3rem 0.3rem rgb(0, 0, 0, 0.15);
+  padding: 2rem 0;
+}
+
+.accordion h2 {
+  text-align: center;
+}
+
+.accordion-items {
+  color: rgb(0, 0, 0, 0.88);
+  margin: 1rem 0;
+}
+
+.accordion-question {
+  min-height: 1.5rem;
+  line-height: 1.25rem;
+  font-size: 0.85em;
+  padding: 1em 1em;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  cursor: pointer;
+  border-bottom: 3px solid rgb(189, 219, 227);
+}
+
+/* creates a plus sign */
+.accordion-question:after {
+  content: "\002B";
+  font-size: 2rem;
+  transition: transform 0.3s ease-in-out;
+}
+
+/* rotates the plus sign on active */
+.accordion-question.active:after {
+  transform: rotate(45deg);
+}
+
+.accordion-answer {
+  display: none;
+  padding: 1.2rem;
+  line-height: 1.2rem;
+  font-style: italic;
+  border-top: 3px solid rgb(189, 219, 227);
+}
+
+/* shows the sibling immediately next to question if question is active */
+.accordion-question.active + .accordion-answer {
+  display: block;
+}
+
+.form-box {
+  min-height: 80vh;
+  height: auto;
+  max-width: 100%;
+  background-image: url(photos/background.jpg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  padding: 2rem 0;
+}
+
+form h2 {
+  text-align: center;
+  padding-bottom: 1rem;
+}
+
+form {
+  max-width: 65%;
+  width: auto;
+  padding: 2rem;
+  min-height: 62vh;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(249, 249, 251, 0.95);
+  border-radius: 2rem;
+  margin: 2rem auto;
+  box-shadow: 0.3rem 0.3rem rgb(0, 0, 0, 0.15);
+}
+
+label,
+input,
+.form-button {
+  font-size: 1em;
+  font-family: "Merriweather", serif;
+  margin: 0.5rem 0;
+  color: rgb(0, 0, 0, 0.88);
+}
+
+label {
+  display: flex;
+  flex-direction: column;
+}
+
+input {
+  padding: 1rem 0;
+  border-radius: 0.5rem;
+  font-style: italic;
+}
+
+.terms {
+  font-size: 0.75em;
+  display: inline;
+  text-align: center;
+}
+
+.form-button {
+  border: 5px solid rgb(189, 219, 227);
+  border-radius: 0.5rem;
+  background-color: rgb(249, 249, 251);
+  padding: 1rem 0;
+}
+
+footer {
+  max-height: 20%;
+  background-color: hsl(213, 12%, 83%);
+  padding: 0 10vw;
+}
+
+footer h2 {
+  font-family: "Special elite", cursive;
+  padding: 1rem 0;
+}
+
+.address {
+  padding-right: 9rem;
+}
+
+.arrow-up-box {
+  width: 5rem;
+  margin: 0 auto;
+}
+
+a.arrow-up {
+  color: rgb(249, 249, 251);
+  font-size: 2rem;
+  display: flex;
+  justify-content: center;
+}
+
+@media (min-width: 768px) {
+  .header-mobile-video {
+    display: none;
+  }
+
+  .header-desktop-video {
+    display: block;
+  }
```

Comment:
never had the time to do this myself but I am so impressed and inspired by how you have done this. nice!!